<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>List</class>
 <widget class="QMainWindow" name="List">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>todo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="toolbarLayout">
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Status: 0 todo / 0 done</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addItem">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="itemsLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>438</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp_lmao">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionClear_all"/>
    <addaction name="actionExit"/>
    <addaction name="actionSort"/>
    <addaction name="actionRemind_me"/>
    <addaction name="actionExport"/>
   </widget>
   <addaction name="menuHelp_lmao"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClear_all">
   <property name="text">
    <string>Clear all</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSort">
   <property name="text">
    <string>Sort</string>
   </property>
  </action>
  <action name="actionRemind_me">
   <property name="text">
    <string>Remind me</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
